### PR TITLE
FPV-50 add an endpoint to get user's candidate

### PR DESCRIPTION
**Pull request by phamduylong**
Added a route to get user's candidate

@jaakkoiot @Niklasnk @kikani-parth @kendrick-807

### CODE REVIEW

File: routes/userRoutes.js [hunk 1]
```diff
@@ -33,4 +34,40 @@ userRouter.get('/id=:id', authorizedOrdinaryUser, async (req, res) => {
...
+                    message: candidate[0].message,
+                    img: candidate[0].img
+                };
+                logger.info("Candidate found successfully! Details:", result);
+                return res.status(200).send(result);
+            }
+            return res.status(204).send({ });
+        }
+        logger.error("Couldn't find user with id: ", userId);
+        return res.status(400).send({ error: "User Not Found!" });
```
Comment (kendrick-807) [inline routes/userRoutes.js]: Should this be 404 or 400

Comment (phamduylong) [reply routes/userRoutes.js]: https://stackoverflow.com/questions/34312023/http-get-request-status-204-vs-404

Comment (kendrick-807) [reply routes/userRoutes.js]: I'm asking for the status code if the user is not found, but according to the link shouldn't it be 204 instead of 400?

Comment (phamduylong) [reply routes/userRoutes.js]: Oh yeah, sounds right :smile:

**Review by kendrick-807 — APPROVED** [review]

**Pull request merged**